tests/e2e/testcases/subnet/non-sov/local/suite.go

```diff
@@ -277,30 +277,6 @@ var _ = ginkgo.Describe("[Local Subnet non SOV]", ginkgo.Ordered, func() {
 
 		commands.DeleteSubnetConfig(subnetName)
 	})
-
-	ginkgo.It("can list a subnet's validators non SOV", func() {
-		nodeIDs := []string{
-			"NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ",
-			"NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg",
-		}
-
-		commands.CreateSubnetEvmConfigNonSOV(subnetName, utils.SubnetEvmGenesisPath)
-		deployOutput := commands.DeploySubnetLocallyNonSOV(subnetName)
-		_, err := utils.ParseRPCsFromOutput(deployOutput)
-		if err != nil {
-			fmt.Println(deployOutput)
-		}
-		gomega.Expect(err).Should(gomega.BeNil())
-
-		output, err := commands.ListValidators(subnetName, "local")
```

Comment:
I believe this test does not need to be removed, but ListValidators implementation fixed (really `blockchain validators` fixed)